common/build.gradle

```diff
@@ -3,31 +3,16 @@ apply from: '../library/quality/quality.gradle'
 
 android {
     compileSdkVersion compileSdk
-    buildToolsVersion buildTools
 
     defaultConfig {
         minSdkVersion minSdk
         targetSdkVersion targetSdk
         versionCode 1
         versionName "1.0"
     }
-
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt')
-        }
-    }
 }
 
 dependencies {
-    // Common dependencies across database modules. Each module still has to provide an
-    // Architecture Components annotation processor and optionally, test dependencies.
-
-    compile "com.android.support:recyclerview-v7:$supportLibraryVersion"
     // Needed to override play services
-    compile "com.android.support:support-v4:$supportLibraryVersion"
-
-    compile "android.arch.lifecycle:runtime:$architectureVersion"
-    compile "android.arch.lifecycle:extensions:$architectureVersion"
+    api "com.android.support:support-v4:$supportLibraryVersion"
```

Comment:
Thinking back on this, I think we should get rid of the dependency block in the common module and add it to the DB and Firestore modules as an `implementation`. @samtstern what do you think?

Reply:
Done, although it has to be `api` since we expose `RecyclerView.Adapter`

Reply:
You sure? The only reason we need that dependency is to override Play Services... We already include the recyclerview dep.